### ISSUE TITLE
Expose more info link on frontend

### ISSUE THREAD

**Issue by smithmilner** [PeaceGeeksSociety/servicesadvisor-3.0#74]
More Info Link field should be shown in the popup or sidebar

https://www.evernote.com/l/AAiNSnXhh9dH564TvLdPQMVdoGEMiE3Ie6Q

**Comment by PeaceGeeksProjects** [comment]
Not working in test.
1. pop-up does not show it
2. only some details show it
3. when you click on it it goes to AffinityBridge.com

**Comment by smithmilner** [comment]
@PeaceGeeksProjects 

1.  Adding it to the popup was not in the original request. It was supposed to go in the detailed view or the popup. I can add it to both if that's required though. It's working as built.
2. Only some service locations actually use the field. It simply doesn't show if there is no link to go to. E.G. http://test.servicesadvisor.org/#/services/31?sector=Basic%20Needs&hideOthers=false has no more link data so it doesn't show anything.
3. I added affinitybridge.com as test content for the field which you can see http://test.servicesadvisor.org/#/services/27?sector=Basic%20Needs&category=Camp%20Interventions&hideOthers=true Other locations should have their own links. E.G. http://test.servicesadvisor.org/#/services/35?sector=Basic%20Needs&hideOthers=false has a different link but it looks like it was linked to a broken site.

**Comment by PeaceGeeksProjects** [comment]
Tx @smithmilner 
1. it was on the evernote for this request 
![image](https://cloud.githubusercontent.com/assets/16202351/25294772/a58b582c-2695-11e7-8983-17a490bb16b9.png)
2. Understood.
3. When will all the links be fixed so the I can fully test?

**Comment by smithmilner** [comment]
I've added a button to the popup.

**Comment by PeaceGeeksProjects** [comment]
Thanks @Smith, It looks too big in the PopUp.  Can it be just a More Info 'link' like in the pic above?
When will all the links be fixed so the I can fully test?

**Comment by smithmilner** [comment]
I think it's best to make it consistent with the link in the sidebar. But yes, it was too big so I've reduced the size a bit.

The link locations themselves are just content. They can be changed in the backend by editing locations. They are all working in the sense they link to other pages. Some of those other pages are broken though haha. I don't know where they're supposed to link to so I just changed them to http://affinitybridge.com/ for now.
